### DISCUSSION TITLE
Replacing the shell in a tab?

### DISCUSSION THREAD

**Discussion by vefatica** [microsoft/terminal#17024] [General]
`START /B %COMSPEC% & EXIT` used to fail in WT.  Now it doesn't.  Was there a change?  Is doing that reliable?  Thanks!

**Comment by zadjii-msft** [comment]
Hard to know for sure without knowing which build didn't work, and which one does. I don't remember anything off the top of my head that was specific to this scenario. 

I honestly don't know if this is an intentionally supported scenario, but if it worked in conhost, the big picture idea is that it _should_ work in WT.

**Comment by DHowett** [comment]
This was fixed in #14544.

**Comment by vefatica** [comment]
Thanks, gentlemen.  I'm glad the behavior is different.  I had asked specifically about `START /B ... & EXIT` but I can't find that post.  What used to happen in that scenario was that the tab would stay open, with a message about the process terminating (and otherwise useless).
